WorkflowConcurrency/Sources/AsyncOperationWorker.swift

```diff
@@ -60,3 +59,85 @@ public struct AsyncOperationWorker<OutputType>: Worker {
         true
     }
 }
+
+extension RenderContext {
+    /// Creates or updates a child workflow of the given type, performs a render
+    /// pass, and returns the result.
+    ///  
+    /// The receiver is the context with which the workflow will be rendered.
+    ///  
+    /// - Returns: The `Rendering` generated by the workflow.
+    /// `WorkflowType`.
+    /// - Parameters:
+    ///   - keyFile: A string that helps uniquely identifies this workflow.
+    ///   - keyFunction: A string that helps uniquely identifies this workflow.
+    ///   - keyLine: An integer that helps uniquely identifies this workflow.
+    ///   - block: Any async closure or function to run. The return `Action`
+    ///   type must match the context's `WorkflowType`.
+    public func run<Action>(
+        keyFile: StaticString = #file,
+        keyFunction: StaticString = #function,
+        keyLine: Int = #line,
+        block: @escaping () async -> Action
+    ) where Action: WorkflowAction, Action.WorkflowType == WorkflowType
+    {
+        AsyncOperationWorker(block)
+            .running(in: self, key: "\(keyFile).\(keyFunction):\(keyLine)")
+    }
+
+    public func run<Action>(
+        key: String,
+        block: @escaping () async -> Action
+    ) where Action: WorkflowAction, Action.WorkflowType == WorkflowType {
+        AsyncOperationWorker(block)
+            .running(in: self, key: key)
+    }
+
+    public func run<Output, Action>(
+        keyFile: StaticString = #file,
+        keyFunction: StaticString = #function,
+        keyLine: Int = #line,
+        block: @escaping () async -> Output,
+        map: @escaping (Output) -> Action
+    ) where Action: WorkflowAction, Action.WorkflowType == WorkflowType
+    {
+        AsyncOperationWorker(block)
+            .running(in: self, key: "\(keyFile).\(keyFunction):\(keyLine)", outputMap: map)
+    }
+
+    public func run<Action>(
+        keyFile: StaticString = #file,
+        keyFunction: StaticString = #function,
+        keyLine: Int = #line,
+        block: @escaping () async throws -> Action,
+        catch: @escaping (Error) -> Action
+    ) where Action: WorkflowAction, Action.WorkflowType == WorkflowType
+    {
+        AsyncThrowingWorker(block)
+            .running(in: self, key: "\(keyFile).\(keyFunction):\(keyLine)") { result -> Action in
+                switch result {
+                case .success(let output):
+                    return output
+                case .failure(let error):
+                    return `catch`(error)
+                }
+            }
+    }
+
+    public func run<Action>(
```

Comment:
You'll want to document the overloads as well so that Xcode will pick up the docs in each case. Also, you should consider adding some code examples similar to what is on `AsyncThrowingWorker`